tests/functional/centos/7/docker-cluster/vagrant_variables.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+# DEPLOY CONTAINERIZED DAEMONS
+docker: True
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
```

Comment:
I'll have to make the change in ceph-build for this first, but I don't think it'll be much trouble.